documentation/Functions.md

```diff
@@ -1,27 +1,27 @@
 ## Functions 
 The following functions are already implemented:
-* installDevonfwIde
-* restoreDevonfwIde
-* restoreWorkspace
-* changeWorkspace
-* executeCommand
-* installCobiGen
-* cobiGenJava
-* createDevon4jProject
-* buildJava
-* createFile
-* changeFile
-* createFolder
-* cloneRepository
-* runServerJava
-* buildNg
-* npmInstall
-* dockerCompose
-* downloadFile
-* nextKatacodaStep
-* adaptTemplatesCobiGen
-* createDevon4ngProject
-* addSetupScript
+* [installDevonfwIde](https://github.com/devonfw-tutorials/tutorial-compiler/wiki/Functions#installDevonfwIde)
```

Comment:
I am not sure but wouldn't this be easier.
 
`:toc:
:toc-title:
`

 We wouldn't have to link them manually, but I am not sure, because I just copied it from the tutorials wiki.